internal/featuresupport/features_olm_operators.go

```diff
@@ -591,3 +591,37 @@ func (f *OpenShiftAIFeature) getFeatureActiveLevel(cluster *common.Cluster, _ *m
 	}
 	return activeLevelNotActive
 }
+
+// AuthorinoFeature describes the support for the Authorino operator.
+type AuthorinoFeature struct{}
+
+func (f *AuthorinoFeature) New() SupportLevelFeature {
+	return &AuthorinoFeature{}
+}
+
+func (f *AuthorinoFeature) getId() models.FeatureSupportLevelID {
+	return models.FeatureSupportLevelIDAUTHORINO
+}
+
+func (f *AuthorinoFeature) GetName() string {
+	return "Authorino"
+}
+
+func (f *AuthorinoFeature) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
+	return models.SupportLevelDevPreview
+}
+
+func (f *AuthorinoFeature) getIncompatibleArchitectures(_ *string) *[]models.ArchitectureSupportLevelID {
+	return nil
+}
```

Comment:
does it support all arch?

Reply:
Done, added arm architecture as not supported.